ext/xmlrpc/xmlrpc-epi-php.c

```diff
@@ -626,7 +626,7 @@ PHP_FUNCTION(xmlrpc_encode_request)
 	size_t method_len;
 	php_output_options out;
 
-	if (zend_parse_parameters(ZEND_NUM_ARGS(), "s!z|a", &method, &method_len, &vals, &out_opts) == FAILURE) {
+	if (zend_parse_parameters(ZEND_NUM_ARGS(), "s!z|a!", &method, &method_len, &vals, &out_opts) == FAILURE) {
```

Comment:
We should be able to default this to empty array instead.

Reply:
I see it now! Although it's a bit sad that `xmlrpc_server_call_method()`s `$out_opts` is nullable.

Reply:
Why is it necessary to keep that one nullable?

Reply:
I think passing an empty array as 2nd arg to `set_output_options()` should have the same effect as passing `NULL`.

Reply:
Oh, I've just realized that I mixed up the arguments passed to `set_output_options()`, and that's why didn't understand what's going on.